src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -335,64 +360,102 @@ private SuggestedActionSet OrganizeRefactorings(Workspace workspace, CodeRefacto
                 return new SuggestedActionSet(refactoringSuggestedActions.ToImmutable(), SuggestedActionSetPriority.Low);
             }
 
-            public async Task<bool> HasSuggestedActionsAsync(ISuggestedActionCategorySet requestedActionCategories, SnapshotSpan range, CancellationToken cancellationToken)
+            public async Task<bool> HasSuggestedActionsAsync(
+                ISuggestedActionCategorySet requestedActionCategories,
+                SnapshotSpan range,
+                CancellationToken cancellationToken)
             {
-                var view = _textView;
```

Comment:
the reason I hold onto local copy of view is so that when a race happens where view is closed but task was in the middle of running in background thread, it won't crash.